ui/selectors/send.test.js

```diff
@@ -279,78 +248,12 @@ describe('send selectors', () => {
     });
   });
 
-  describe('getSendToAccounts()', () => {
-    it('should return an array including all the users accounts and the address book', () => {
-      expect(getSendToAccounts(mockState)).toStrictEqual([
-        {
-          code: '0x',
-          balance: '0x47c9d71831c76efe',
-          nonce: '0x1b',
-          address: '0xfdea65c8e26263f6d9a1b5de9555d2931a33b825',
-          name: 'Send Account 1',
-        },
-        {
-          code: '0x',
-          balance: '0x37452b1315889f80',
-          nonce: '0xa',
-          address: '0xc5b8dbac4c1d3f152cdeb400e2313f309c410acb',
-          name: 'Send Account 2',
-        },
-        {
-          code: '0x',
-          balance: '0x30c9d71831c76efe',
-          nonce: '0x1c',
-          address: '0x2f8d4a878cfa04a6e60d46362f5644deab66572d',
-          name: 'Send Account 3',
-        },
-        {
-          code: '0x',
-          balance: '0x0',
-          nonce: '0x0',
-          address: '0xd85a4b6a394794842887b8284293d69163007bbb',
-          name: 'Send Account 4',
-        },
-        {
-          address: '0x06195827297c7a80a443b6894d3bdb8824b43896',
-          name: 'Address Book Account 1',
-          chainId: '0x3',
-        },
-      ]);
-    });
-  });
-
   describe('getTokenBalance()', () => {
     it('should', () => {
       expect(getTokenBalance(mockState)).toStrictEqual(3434);
     });
   });
 
-  describe('getUnapprovedTxs()', () => {
```

Comment:
Were all of these tests unhelpful, or could/should they be moved elsewhere?

Reply:
Good call! Moved them to the metamask duck test file